CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - PR #2906: Moving `linalg` decomp to RAFT namespaces
 - PR #2996: Removing the max_depth restriction for switching to the batched backend
 - PR #3004: Remove Single Process Multi GPU (SPMG) code
+- PR #3032: FIL: Add optimization parameter `blocks_per_sm` for all but tiny models
```

Comment:
I will update changelog once the other changes are known :)

Reply:
I haven't seen any code changes that exclude "tiny" models.

Reply:
the code will permit those, but this feature will not speed up the tiniest models. I was hoping for people who read changelogs to make a quick decision of whether it's likely to help them. Should it still go away?